HSLdevcom/jore-map-ui
# 1102 stop creation outside of local area

Closes #1102 

BE PR: https://gitlab.hsl.fi/jore/jore-map-backend/merge_requests/158

Flow described in card

Added idSuffix to INode. This way code is a lot cleaner (no explicit code for handling idSuffix is needed). Just have to make sure that when node is created, final nodeId saved as id + suffix

How to test:

* Try to create a new node in Helsinki
    * try to switch transitType -> it should re-render nodeView and generates a new nodeId
* Try to create a new node in Kouvola
    * notification should be shown that node id automatic generation has failed
    * input 5 letters, select ending from dropdown

There is some nodeId input lag. Will be fixed here: https://github.com/HSLdevcom/jore-map-ui/issues/1201
